lua/orgmode/config/defaults.lua

```diff
@@ -87,6 +94,7 @@ return {
       org_todo_prev = 'ciT',
       org_toggle_checkbox = '<C-Space>',
       org_open_at_point = '<Leader>oo',
+      org_edit_src_block = '<Leader>ose',
```

Comment:
Lets name this `org_edit_special` since it's called like that in emacs orgmode.

Reply:
Sounds good, I'll be more careful to check the _Emacs_ options in the future.